src/main/java/com/knighttodo/knighttodo/rest/RoutineResource.java

```diff
@@ -78,9 +78,9 @@
     }
 
     @DeleteMapping("/{routineId}")
-    public ResponseEntity<Void> deleteRoutine(@PathVariable String routineId) {
+    public ResponseEntity<Void> deleteRoutine(@PathVariable String blockId, @PathVariable String routineId) {
         log.info("Rest request to delete routine by id : {}", routineId);
-        routineService.deleteById(routineId);
+        routineService.deleteById(blockId, routineId);
         return ResponseEntity.ok().build();
     }
 }
```

Comment:
There is misscommunication between service and resource, cause service method accepts only one parameter to deleteById

Reply:
fixed it. Thanks.